backend/middlewares/sessionMiddleware.js

```
@@ -1,12 +1,13 @@
-const sessionMiddleware = async (req, res, next)=>{
-    console.log(req.session.user);
-    
-    
-    if (req.session.user !== undefined) {
-        next();
+const sessionMiddleware = (req, res, next) => {
+    if (req.session.user && req.session.user.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)